x-pack/test/security_solution_endpoint/apps/endpoint/resolver.ts

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getPageObjects, getService }: FtrProviderContext) {
+  const pageObjects = getPageObjects(['common', 'timePicker', 'hosts', 'settings']);
+  const testSubjects = getService('testSubjects');
+  const esArchiver = getService('esArchiver');
+
+  describe('Endpoint Event Resolver', function () {
+    this.tags(['ciGroup7']);
```

Comment:
I don't think you need to add this tag since it's already in the index https://github.com/elastic/kibana/blob/master/x-pack/test/security_solution_endpoint/apps/endpoint/index.ts#L17